CATs/Security Group.cat.rb

```diff
@@ -0,0 +1,224 @@
+# Copyright 2017 RightScale
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name 'Security Group'
+rs_ca_ver 20160622
+short_description "![logo](https://cdn0.iconfinder.com/data/icons/flat-security-icons/512/lock-open-blue.png =64x64)\n
+Provisions and manages an SDN-based Security Group."
+long_description 'Provisions and manages an SDN-based Security Group within the chosen cloud.'
+
+###
+# User inputs
+###
+parameter 'cloud' do
+  label 'Cloud'
+  category 'Location'
+  default 'EC2 us-east-1'
+  description "json:{\"definition\":\"get_cloud_names\", \"description\": \"The cloud the security group will reside in.\"}"
```

Comment:
What is the meaning of this json syntax? I don't think SelfService parses this, and it's not immediately obvious to me what this means.